settings.go

```diff
@@ -20,8 +20,10 @@ type CallToActionsSetting struct {
 
 // CallToActionsItem contains Get Started button or item of Persist Menu
 type CallToActionsItem struct {
-	Type    string `json:"type,omitempty"`
-	Title   string `json:"title,omitempty"`
-	Payload string `json:"payload,omitempty"`
-	URL     string `json:"url,omitempty"`
+	Type               string `json:"type,omitempty"`
+	Title              string `json:"title,omitempty"`
+	Payload            string `json:"payload,omitempty"`
+	URL                string `json:"url,omitempty"`
+	WebviewHeightRatio string `json:"webview_height_ratio,omitempty"`
```

Comment:
You mention in your comment about there being only three different values which are currently valid here. Personally, I think this warrants use of a constant.

I'd go for something like:

```go
const (
  WebviewCompact = iota
  WebviewFull
  WebviewTall
)
```